R/install.R

```diff
@@ -235,20 +235,30 @@ lantern_install_libs <- function(version, type, install_path, install_config) {
 }
 
 install_type_windows <- function(version) {
+  verbose <- Sys.getenv("TORCH_VERBOSE_INSTALL")
   cuda_version <- NULL
   cuda_path <- Sys.getenv("CUDA_PATH")
 
   if (nzchar(cuda_path)) {
-    versions_file <- file.path(cuda_path, "version.txt")
+    if (verbose == "1") { message(paste0("CUDA_PATH environment variable is set to ", cuda_path)) }
+    versions_file <- file.path(cuda_path, "version.json")
     if (file.exists(versions_file)) {
-      cuda_version <- gsub("CUDA Version |\\.[0-9]+$", "", readLines(versions_file))
+      if (verbose == "1") { message(paste0(cuda_path, "\\version.json found, extracting CUDA SDK version from it.")) }
+      cuda_version <- stringr::str_extract(readLines(versions_file, n=4)[4], "[0-9]+.[0-9]+")
```

Comment:
I don't think we want to take a `stringr` dependency here. Can we rewrite with base R?